src/sleepecg/io/nsrr.py

```diff
@@ -61,8 +63,19 @@ def _get_nsrr_url(db_slug: str) -> str:
     str
         The download URL.
     """
+    global _nsrr_token
     if _nsrr_token is None:
-        raise RuntimeError("NSRR token not set, use `sleepecg.set_nsrr_token(<token>)`!")
+        _nsrr_token = os.environ.get("NSRR_TOKEN")
+        if (
+            _nsrr_token := _nsrr_token
+            or os.environ.get("NSRR_TOKEN")
+            or get_config_value("nsrr_token")
+        ) is None:
+            raise RuntimeError(
+                "NSRR token not set, use `sleepecg.set_nsrr_token(<token>)`, set the "
+                "token in the 'config.yml' file or set an environment variable "
+                "'NSRR_TOKEN'!"
+            )
```

Comment:
```suggestion
    if (
        _nsrr_token := _nsrr_token
        or os.environ.get("NSRR_TOKEN")
        or get_config_value("nsrr_token")
    ) is None:
        raise RuntimeError(
            "NSRR token not set, use `sleepecg.set_nsrr_token(<token>)`, set the token "
            "in the configuration, or set an environment variable NSRR_TOKEN."
        )
```

Reply:
I cannot automatically apply this suggestion, I get the error "Applying suggestions on deleted lines is not supported." I added this change manually.